src/messages/ru/hipanel:client.php

```diff
@@ -263,6 +264,8 @@
     'Primary currency for invoices' => 'Основная валюта для инвойсов',
     'Exchange all debts to primary currency' => 'Переводить все долги в основную валюту',
     'When the primary currency (say EUR) balance is positive and the secondary currency (say USD) has debts, exchange as much available EUR as possible to close USD debts' => 'При наличии позитивного баланса в основной валюте (допустим EUR), и долга во вторичной валюте (скажем USD), обменивать как можно больше доступных EUR чтобы покрыть долг в USD',
+    'Exchange all debs to primary currency including prepayments for expected resource consumption' => 'Переводить все долги в основную валюту, включая предоплату за переборы',
```

Comment:
Exchange prepayments for the expected resources consumption
Обменивать предоплату за ожидаемые переборы ресурсов

Когда включён автоматический обмен валют для закрытия долгов, этот флаг указывает, что внесённая предоплата за перебор ресурсов будет сконвертирована в основную валюту сразу после поступления на счёт. Это может быть полезно для избежания образования задолженности на основном счету, в то время как на вторичном будет оставаться неиспользованный аванс.

When "exchange currency for debts automatically" is enabled, this flag indicates that prepayments for the expected resources consumption SHOULD be converted to primary currency immediately. It can be useful to prevent debts in primary currency while the secondary currency balance is positive due to prepayment.